src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Verbs.cs

```diff
@@ -475,4 +475,38 @@ internal partial class UpdateVerbParser
         }
     }
 
+    internal partial class ConfigVerbParser
+    {
+        internal static void Register(CommandLineApplication app,
+                                      Func<ILogger> getLogger)
+        {
+            app.Command("config", ConfigCmd =>
+            {
+                ConfigCmd.Command("paths", PathsCmd =>
+                {
+                    CommandArgument workingdirectory = PathsCmd.Argument(
+                        "working-directory", Strings.ConfigPathsWorkingDirectoryDescription);
```

Comment:
```suggestion
                    CommandArgument workingdirectory = PathsCmd.Argument(
                        "working-directory",
                        Strings.ConfigPathsWorkingDirectoryDescription);
```

Reply:
Hi Jeff! At the top of this file, it says:
> Do not manually edit this autogenerated file: instead modify the neighboring .tt file (text template) and/or NuGet.CommandLine.Xplat\Commands\Commands.xml (data file), then re-execute the text template via "run custom tool" on VS context menu for .tt file, or via dotnet-t4 global tool.
Given this, is it OK for me to directly make these changes?

Reply:
Dang, I guess in a future commit we can update the `.tt` file to use better formatting 😞